multinet/swagger/template.yaml

```
@@ -173,6 +173,15 @@ parameters:
       type: string
       example: key0
 
+  upload_id:
+    name: upload_id
+    in: path
+    description: ArangoDB key for multipart upload file document
+    required: true
+    schema:
+      type: integer
+      example: 50841
+
```

I don't feel strongly about what I put in the description, just wanted to update the language to refer to collections instead of documents

```suggestion
  upload_id:
    name: upload_id
    in: path
    description: The collection ID associated with this upload
    required: true
    schema:
      type: string
      example: u-1234abcd

```